unified-codes/apps/web/src/components/containers/explorer/ExplorerSearchBar.tsx

```diff
@@ -13,7 +13,7 @@ const useStyles = makeStyles((theme: ITheme) =>
   createStyles({
     root: { color: theme.palette.text.primary },
     input: { paddingLeft: 15, color: '#a2a2a2' },
-    button: { marginTop: 15 },
+    button: { marginTop: 15, minWidth: 0 },
```

Comment:
I couldn't see the impact of specifying a zero min-width. Is this required?

Reply:
Oh - may have been a bit pedantic of me hehe - noticed some weirdness (default minWidth is set to 64):
![image](https://user-images.githubusercontent.com/65875762/129994800-61b7d02a-6824-4808-810e-f4c258d15114.png)

vs (which is still a bit weird but better?)

![image](https://user-images.githubusercontent.com/65875762/129995113-dff937fe-1f32-4d77-b1db-b304ad7c58c8.png)